tekton-catalog/pipeline-loops/cmd/webhook/main.go

```diff
@@ -66,7 +65,7 @@ func newDefaultingAdmissionController(ctx context.Context, cmw configmap.Watcher
 
 		// A function that infuses the context passed to Validate/SetDefaults with custom metadata.
 		func(ctx context.Context) context.Context {
-			return contexts.WithUpgradeViaDefaulting(store.ToContext(ctx))
+			return store.ToContext(ctx)
```

Comment:
Do you know what version of Kubernetes that we need if we are going to remove this? I remember `WithUpgradeViaDefaulting` is to convert the old context to new context.

Reply:
This change is coming from the upstream tekton upgrade. As I understood, if we want to upgrade to support retries, we will have to upgrade this as well. I have tested with 1.21.